packages/kubernetes-1.15/kubernetes-1.15.spec

```
@@ -79,6 +80,7 @@ install -m 0644 %{S:2} %{buildroot}%{_cross_templatedir}/kubelet-env
 install -m 0644 %{S:3} %{buildroot}%{_cross_templatedir}/kubelet-config
 install -m 0644 %{S:4} %{buildroot}%{_cross_templatedir}/kubelet-kubeconfig
 install -m 0644 %{S:5} %{buildroot}%{_cross_templatedir}/kubernetes-ca-crt
+install -m 0644 %{S:6} %{buildroot}%{_cross_templatedir}/kubelet-sysctl.conf
```

The template directory is for templates that we render with settings.

In order for the sysctl.conf drop-in to be used directly, you'll want something like this instead:
```suggestion

mkdir -p %{buildroot}%{_cross_sysctldir}
install -p -m 0644 %{S:6} %{buildroot}%{_cross_sysctldir}/90-kubelet.conf
```